README.md

```diff
@@ -1,2 +1,40 @@
-# Portfolio
- A mobile Portfolio project to showcase my skills and tell a little about myself
+## Porfolio-mobile
+> This is a portfolio that shows my proficiency as I go through Microverse. It is a work-in-progress and I hope to make it a masterpiece.
+
+
+## Built With
+- HTML
+- CSS
+- Linter
+
```

Comment:
🥳  Great job, on adding a custom README to make it professional but guess what, there are some tiny tweaks we can add to make it look even better. Have a look at the comments in the file.

- [ ] Some recruiters may not be too technical and might need help on how to have the app cloned locally. Let's provide a brief walkthrough of the process here. If you need inspiration have a look at the image below
![image](https://user-images.githubusercontent.com/29071984/170604367-7d7b5ab4-c5a0-4914-9b4b-e0f73ed42b16.png)